migrations/versions/9f7a13ec4cef_.py

```diff
@@ -0,0 +1,46 @@
+"""empty message
+
+Revision ID: 9f7a13ec4cef
+Revises: 90f976272da9
+Create Date: 2018-01-08 11:27:42.117647
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+import json
+
+from models import db, Node
+from app import app
+
+revision = '9f7a13ec4cef'
+down_revision = '90f976272da9'
+branch_labels = None
+depends_on = None
+
+db.init_app(app)
+
+
+def upgrade():
+    for node in Node.query.all():
```

Comment:
This migration clear values from column `connection_config`.
Instead of complicated migration, I would select like this: `SELECT * from node WHERE proposal IS NOT NULL`

Reply:
This would not handle case, where `proposal` is a valid `json`, but it does not contain `'service_proposal'` key. This is why dev environment is failing.